### PR TITLE
README refresher and updating demo page

### PR DESCRIPTION
**Pull request by lawong-d2l**
Not Urgent
In an effort for an easier hand over, I want to make sure the demos and docs are relativity useful.
Also allows `polymer serve --open` to go directly to the demo page

### CODE REVIEW

**Comment by lawong-d2l** [comment]
Preview of the new demo page
image

**Comment by jwong-d2l** [comment]
lint

File: demo/index.html [hunk 1]
```diff
@@ -4,23 +4,95 @@
...
+			import '@polymer/iron-demo-helpers/demo-snippet';
+			import 'd2l-icons/d2l-icons.js';
+			import 'd2l-icons/tier2-icons.js';
+			import 'd2l-icons/tier3-icons.js';
+			import "../components/d2l-sequences-module-name.js";
+			import "../components/d2l-sequences-topic-name.js";
+			import "../components/d2l-sequences-iterator.js";
+			import "../components/d2l-sequences-content-router.js";
+			const $_documentContainer = document.createElement('template');
+			$_documentContainer.innerHTML = `
```
Comment (jwong-d2l) [inline demo/index.html]: can you get rid of all this innerHTML garbage too, like I did for https://github.com/Brightspace/sequence-launcher-fra/pull/59/files#diff-3e22c50371c76d64ea9738dbc46a6e57

File: package-lock.json [hunk 2]
```diff
@@ -1484,10 +1484,10 @@
       "requires": {
         "@polymer/iron-media-query": "^3.0.0",
         "@polymer/polymer": "^3.0.0",
-        "d2l-colors": "github:BrightspaceUI/colors#semver:^4",
```
Comment (jwong-d2l) [inline package-lock.json]: update your version of npm too.  you can't un-semver all these things.

**Pull request merged**